platforms/documentation/docs/src/docs/userguide/dep-man/declaring-depedencies/declaring_dependencies_basics.adoc

```diff
@@ -0,0 +1,221 @@
+// Copyright (C) 2023 Gradle, Inc.
+//
+// Licensed under the Creative Commons Attribution-Noncommercial-ShareAlike 4.0 International License.;
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      https://creativecommons.org/licenses/by-nc-sa/4.0/
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+[[declaring-dependencies-basics]]
+= Declaring Dependencies Basics
+
+[[sec:dependency-types]]
+== Types of dependencies
+
+There are three main types of dependencies in Gradle:
+
+1. *Module Dependencies*: Refer to libraries from external repositories.
+2. *Project Dependencies*: Refer to other projects in the same multi-project build.
+3. *File Dependencies*: Refer to local files or directories, such as `.jar` or `.aar` files.
+
+[[sub:module_dependencies]]
+=== 1. Module dependencies
+
+Module dependencies are the most common dependencies.
+They refer to a dependency that is identified by module coordinates (group, name, and version):
+
+====
+include::sample[dir="snippets/artifacts/externalDependencies/kotlin",files="build.gradle.kts[tags=module-dependencies]"]
+include::sample[dir="snippets/artifacts/externalDependencies/groovy",files="build.gradle[tags=module-dependencies]"]
+====
+
+Gradle offers multiple notations for declaring module dependencies, including string notation and map notation.
+
+- **String Notation:** Simplifies dependency declaration by combining the group, name, and version into a single string.
+- **Map Notation:** Allows for a more detailed specification, enabling the definition of all properties.
+
+For advanced configurations, such as making a dependency transitive, you can use closures alongside these notations.
+
+Refer to the link:{groovyDslPath}/org.gradle.api.artifacts.dsl.DependencyHandler.html[`DependencyHandler`] class in the API documentation for more examples and details.
+Additionally, explore the link:{javadocPath}/org/gradle/api/artifacts/ExternalModuleDependency.html[`ExternalModuleDependency`] class to understand the full range of properties and configuration methods available.
```

Comment:
💭 Is there a userguide page that goes into detail on these that would be a better link target?

Reply:
I removed this since this is the basics and they should not be getting plastered with links so early.